src/MainWindow.vala

```diff
@@ -214,11 +214,40 @@ public class AppCenter.MainWindow : Hdy.ApplicationWindow {
 
         spinner = new Gtk.Spinner ();
 
+        var automatically_install_flatpak_updates_button = new Granite.SwitchModelButton (_("Automatic Updates")) {
+            description = _("Automatically install updates to curated apps")
+        };
+
+        var menu_popover_grid = new Gtk.Grid () {
+            column_spacing = 6,
+            margin_bottom = 6,
+            margin_top = 12,
+            orientation = Gtk.Orientation.VERTICAL,
+            row_spacing = 6
+        };
+
+        menu_popover_grid.add (automatically_install_flatpak_updates_button);
+
+        menu_popover_grid.show_all ();
+
+        var menu_popover = new Gtk.Popover (null);
+        menu_popover.add (menu_popover_grid);
+
+        var menu_button = new Gtk.MenuButton () {
+            can_focus = false,
+            image = new Gtk.Image.from_icon_name ("open-menu", Gtk.IconSize.SMALL_TOOLBAR),
```

Comment:
This should be the large toolbar size, but I know that's gonna require some styling fixes as well since we do weird things with margins/padding in CSS here (I think due to the updates badge overlay). I may have started addressing it in #1230 but I also know that PR is pretty outdated.

Reply:
Any suggestions on how we are going to implement this at this point?

Reply:
@meisenzahl actually in testing, using `LARGE_TOOLBAR` looks really close; we might just have to tweak the styling here to match other apps; it looks like 3px matches Files, at least, but I'm also not sure how we would get it to scale with the text-size like [our `rem()` sass function](https://github.com/elementary/stylesheet/blob/9b7122b07b3a1878501d13d842c54e83ef9061c6/src/gtk-3.0/_index.scss#L162-L178) in the stylesheet… :thinking: That might be over-thinking it, though, and 3px here might be fine.

https://github.com/elementary/appcenter/blob/c9d9f16e4617160d511ba099b0fd8aa29dbea98e/data/styles/application.css#L18-L21

Reply:
Using `LARGE_TOOLBAR` and 0.188rem for padding:

![Bildschirmfoto von 2021-12-13 20 35 43](https://user-images.githubusercontent.com/10796736/145876694-f5023d80-f296-407b-bb43-bea6b4cb4ee8.png)